GPTsettings.py

```diff
@@ -49,10 +50,9 @@ def a_func(a:int, b:int):
     Some desired properties:
         - When specifying the positions, Use the right line for the comment as specified in the diff
         - Be really careful with this! Don't use the ' token in your comments (only when needed for the code) or it will break the python code.
-        - Provide code suggestions as much as possible in your comments, for that you can use the standart format ```suggestion <suggested_change> ```
+        - Provide code suggestions as much as possible in your comments, for that you can use the standart format ```suggestion \n <suggested_change> \n```
```

Comment:
Avoid placing sensitive logic in string literals and ensure to handle exceptions gracefully without exposing stack traces to the end-user.